python_modules/dagster/dagster/_core/definitions/asset_health/asset_check_health.py

```diff
@@ -203,6 +203,11 @@ async def get_asset_check_status_and_metadata(
             context,
         )
 
+        if not context.asset_graph.has(
```

Comment:
i'm a bit confused. wouldn't this need to be above `await AssetCheckHealthState.compute_for_asset_checks`? the rest of the fn is just getting the lengths of lists in `asset_check_health_state`.

Reply:
oh oops, I meant to move just the comment up. One moment